src/server/routes/robots.txt.ts

```diff
@@ -0,0 +1,6 @@
+export default defineEventHandler(async (event) => {
```

Comment:
I think this is a file that could be created during build-time. But this works too.

Reply:
It actually gets created during build time if we use generate